debugger/Dockerfile

```diff
@@ -2,6 +2,9 @@ FROM ubuntu
 
 ENV DEBIAN_FRONTEND noninteractive
 
+RUN rm -rf /var/lib/apt/lists/*
+RUN apt-get clean
+RUN apt-get update -o Acquire::CompressionTypes::Order::=gz
```

Comment:
Ha, why we need this lol

Reply:
no idea.....is this to do w/ deployment?